### PR TITLE
SEO-2024 Blazor Split Button External Linking

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @ChristoIssac ,

Please review the changes made in Blazor Split Button External Linking changes.

Regards,
Yvone.

### CODE REVIEW

**Review by Girija-Rajendran — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,15 +1,62 @@
...
+
+### About Syncfusion
+
+Founded in 2001 and headquartered in Research Triangle Park, N.C., Syncfusion has more than 27,000 customers and more than 1 million users, including large financial institutions, Fortune 500 companies, and global IT consultancies.
```
Comment (ChristoIssac) [inline README.md]: Change the 27,000 to 28,000

File: README.md [hunk 2]
```diff
@@ -1,15 +1,62 @@
...
+
+<hr style="height:0.3px;border:none;color:lightgrey;background-color:lightgrey;" />
+
+<p align="center">
+<a href="mailto:sales@syncfusion.com?Subject=Syncfusion Blazor Split Buttons - GitHub" target="_top">sales@syncfusion.com</a> | <a href="https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-buttons-github-samples">www.syncfusion.com</a>| Toll Free: 1-888-9 DOTNET <br>
```
Comment (ChristoIssac) [inline README.md]: image
Remove the `s`

**Review by ChristoIssac — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Yvone-Atieno) [reply README.md]: Changed

File: README.md (hunk 2 above)
Comment (Yvone-Atieno) [reply README.md]: Removed

**Review by ChristoIssac — APPROVED** [review]

**Pull request merged**